scripts/cluster.sh

```diff
@@ -215,6 +215,329 @@ function download_iso() {
     log "INFO" "ISO downloaded successfully to ${ISO_FOLDER}"
 }
 
+function get_iso() {
+    # Unified function that handles ISOs for both master and worker nodes
+    # Parameters:
+    #   $1 - cluster_name: Name of the cluster
+    #   $2 - node_type: "master" or "worker"
+    #   $3 - action: "url" or "download"
+    #   $4 - download_path: Path to download ISO (if action is "download")
+    #   $5 - iso_type: "minimal" or "full", defaults to "minimal"
+    
+    local cluster_name="$1"
+    local node_type="$2"
+    local action="$3"
+    local download_path="$4"
+    local iso_type="${5:-minimal}"  # Default to minimal if not specified
+    local day2_cluster_name="${cluster_name}-day2"
+    
+    # Export for use in other functions
+    export CLUSTER_NAME="${cluster_name}"
+    export ISO_FOLDER="${download_path:-${ISO_FOLDER}}"
+    export ISO_TYPE="${iso_type}"
+    
+    log "INFO" "ISO request: cluster=${cluster_name}, type=${node_type}, action=${action}, path=${download_path}, iso_type=${iso_type}"
+    
+    if [[ "${node_type}" != "master" && "${node_type}" != "worker" ]]; then
+        log "ERROR" "Invalid node type: ${node_type}. Must be 'master' or 'worker'."
+        exit 1
+    fi
+    
+    if [[ "${action}" != "url" && "${action}" != "download" ]]; then
+        log "ERROR" "Invalid action: ${action}. Must be 'url' or 'download'."
+        exit 1
+    fi
+    
+    if [[ "${iso_type}" != "minimal" && "${iso_type}" != "full" ]]; then
+        log "ERROR" "Invalid ISO type: ${iso_type}. Must be 'minimal' or 'full'."
+        exit 1
+    fi
+    
+    # Color codes for output formatting
+    local YELLOW='\033[1;33m'
+    local GREEN='\033[1;32m'
+    local BOLD='\033[1m'
+    local NC='\033[0m' # No Color
+    
+    # Worker nodes require a day2 cluster and different ISO handling
+    if [ "${node_type}" = "worker" ]; then
+        # For worker nodes, we need to ensure we have a day2 cluster first
+        _ensure_day2_cluster "${cluster_name}" "${iso_type}"
+        
+        if [ "${action}" = "url" ]; then
+            # Get the ISO URL directly from aicli
+            log "INFO" "Getting ISO URL for day2 cluster ${day2_cluster_name}..."
+            local iso_url=$(aicli info iso "${day2_cluster_name}" 2>/dev/null | grep -i "url" | head -1 | sed 's/.*: //' | tr -d ' ' || echo "")
+            
+            # If we couldn't get the URL, provide alternative instructions
+            if [ -z "${iso_url}" ]; then
+                log "WARNING" "Could not get direct ISO URL from aicli info iso command"
+                local console_url="https://console.redhat.com/openshift/assisted-installer/clusters"
+                
+                # Display URL with formatting
+                echo ""
```

Comment:
why all this formatting is required? This creates tons of text for which exactly purpose? Lets just pring simple text without all the alternatives. Simplify it please